src/utils/authRedirectRoute.jsx

```diff
@@ -0,0 +1,27 @@
+import { useAuth } from "./useAuth";
```

Comment:
Maybe change the name to something with like alreadyLoggedIn or something like that in it. I wasn't really sure what an authRedirect route would mean at first.

Reply:
I just changed it to AlreadyLoggedInRedirect